module/data/abstract.mjs

```diff
@@ -56,7 +56,23 @@ export default class SystemDataModel extends foundry.abstract.DataModel {
    * @returns {object}  Fully merged schema.
```

Comment:
I guess this should actually be `@returns {DataSchema}`, as well as the two parameters above it that GH won't let me highlight.